src/consensus.rs

```diff
@@ -160,6 +163,7 @@ where
             repair,
             cancel_token,
             votor_handle,
+            txs_receiver,
```

Comment:
still not in love with the name `receiver` for a network interface / socket, but we can also keep it if we don't immediately find something better

Reply:
Yes, I cannot think of a better name right now either.  I somehow think that `receiver` makes a bit of sense as will only ever receive msgs here